src/main/java/eu/darkbot/api/objects/Gui.java

```diff
@@ -0,0 +1,29 @@
+package eu.darkbot.api.objects;
+
+public interface Gui extends Rectangle {
```

Comment:
i am not convinced that gui (point on screen) should mix with rectangle/shapes (position on the map)